lib/honeybadger/utils.ex

```diff
@@ -14,4 +14,17 @@ defmodule Honeybadger.Utils do
     |> tl
     |> Enum.join(".")
   end
+
+  @doc """
+  Resolve environment name from Application configuration or from Mix.env
+  """
+  def environment_name do
+    case Application.get_env(:honeybadger, :mix_env, nil) do
+      nil ->
+        Mix.env
```

Comment:
Per #8 I'm pretty sure we shouldn't be relying on `Mix` at runtime. @rbishop can you confirm when you have the chance?